javascript/lib/api/src/client/constants/header.ts

```diff
@@ -0,0 +1,16 @@
+/*!
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+export enum Header {
```

Comment:
Nice that you added this enum 👍
I however would have expected that the existing code is then also adjusted to use the enum.
That should hopefully, using "search/replace" in the IDE, not be too much work.
Could you please do that?

Reply:
done